source/SIL.AppBuilder.Portal.Frontend/src/ui/components/user-table/data.tsx

```diff
@@ -70,11 +67,27 @@ export function withData(WrappedComponent) {
 
     toggleLock = async (user) => {
 
-      const { updateStore } = this.props;
+      const { updateStore, t } = this.props;
+
+      const successMessage = !user.attributes.isLocked ?
+        t('users.operations.lock.success') :
+        t('users.operations.unlock.success');
+
+      const errorMessage = !user.attributes.isLocked ?
```

Comment:
the logic is weird (generally, ternaries shouldn't be negative (positive thoughts!))

The way I read this:

 - If the user is _not_ locked,
   - the error message is _lock_.error
 - otherwise, if the user is _locked_,
   - the error message is the _unlocked_ error

just seems like this is semantically confusing / not matching?